docs/understanding-radicle/faq.md

```diff
@@ -142,6 +142,16 @@ GitHub repository.
 For help, join our #support channel in our Matrix chat or in the Help category
 of our Discourse.
 
+## How do I join your Matrix channel?
```

Comment:
Somehow this feels like there should be a dedicated community section in the docs. Feels hidden in the FAQs. We can do it in a follow-up.